ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/participantProfileServiceAgent.service.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { TestBed } from '@angular/core/testing';
+import { HttpClient } from '@angular/common/http';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { forkJoin, of } from 'rxjs';
+import { TranslateService } from '@ngx-translate/core';
+import { ConfigurationService, LoggingService, ParticipantProfileServiceAgent, SessionMementoService, UserProfile } from 'ddp-sdk';
+
+describe('ParticipantProfileServiceAgent', () => {
+    let service: ParticipantProfileServiceAgent;
+    let httpClient: HttpClient;
+    let httpTestingController: HttpTestingController;
+    let config: ConfigurationService;
+    const backendUrl = 'https://pepper-dev.datadonationplatform.org';
+
+    beforeEach(() => {
+        config = new ConfigurationService();
+        config.backendUrl = backendUrl;
+        config.studyGuid = 'ATCP';
+        const loggingServiceSpy: jasmine.SpyObj<LoggingService> = jasmine.createSpyObj('LoggingService', ['logException']);
+        const sessionSpy = new SessionMementoService({} as TranslateService, config);
+        spyOnProperty(sessionSpy, 'sessionObservable').and.returnValue(of({ participantGuid: 'USERGUID' }));
+
+        TestBed.configureTestingModule({
+            imports: [HttpClientTestingModule]
```

Comment:
**# Optional**
We could also have TestBed setup this way:
```
        TestBed.configureTestingModule({
            imports: [HttpClientTestingModule],
            providers: [
                ParticipantProfileServiceAgent,
                {
                    provide: SessionMementoService,
                    useValue: sessionSpy
                },
                {
                    provide: 'ddp.config',
                    useValue: config
                },
                {
                    provide: LoggingService,
                    useValue: loggingServiceSpy
                }
            ]
        });
        httpTestingController = TestBed.inject(HttpTestingController);
        service = TestBed.inject(ParticipantProfileServiceAgent);
```

In order to avoid the unnecessary import of `HttpClient`, as it's already provided by `HttpClientTestingModule`.

It's totally up to you, though.